### PR TITLE
Bump versions & update CHANGELOG

### PR DESCRIPTION
**Pull request by Yatekii**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -31,25 +40,21 @@ An example is this config:
...
-## [0.8.0]
-
-### Added
-
-- Add Windows support with the help of crossterm instead of termion.
-
-### Changed
-
-### Fixed
+- Panics in app that could occur due to a bug will no longer mess up the users terminal.
```
Comment (dbrgn) [inline CHANGELOG.md:55]:
```suggestion
- Panics in app that could occur due to a bug will no longer mess up the user's terminal.
```

**Comment by dbrgn** [comment]
Also needs some probe-rs 0.8.0 API updates.

**Comment by Yatekii** [comment]
> Also needs some probe-rs 0.8.0 API updates.

Nope, this one should be good, but the CI built before I released probe-rs:0.8.0 so it couldn't find it on crates.io :) Same goes for probe-rs-rtt:0.3.0 which we hopefully can release tomorrow.

**Comment by dbrgn** [comment]
I don't think so, there's a `core.reset_and_halt` call in `main.rs` that wants a timeout :slightly_smiling_face:

**Comment by Yatekii** [comment]
I don't understand ... I built this against probe-rs master so many times? oO

**Pull request merged**